src/ilasm/asmman.cpp

```diff
@@ -121,8 +121,9 @@ AsmManAssembly*     AsmMan::GetAsmRefByName(__in __nullterminated const char* sz
         //ret = m_AsmRefLst.FIND(&X);
         //X.szAlias = NULL;
         DWORD L = (DWORD)strlen(szAsmRefName);
-        for(int i=0; (ret = m_AsmRefLst.PEEK(i))&&
-            ((ret->dwAlias != L)||strcmp(ret->szAlias,szAsmRefName)); i++);
+        for (int i = 0; (ret = m_AsmRefLst.PEEK(i)); i++) {
+            if (((ret->dwAlias == L) && !strcmp(ret->szAlias, szAsmRefName))) break;
+        }
```

Comment:
Is this fixing something?  It doesn't appear to be changing behavior, just cleaning up terrible style.  If this isn't related to the fix, it should be very clearly called out in the PR description of the changes.  In general, we prefer not to combine formatting/style changes with behavioral changes, but it's acceptable in some cases if you're changing nearby code anyway.